packages/common/src/member-list/member-list.tsx

```diff
@@ -32,7 +30,7 @@ export interface MemberListProps {
  * into it. Custom memberRenderer can be used to modify how the users are rendered. For example
  * you can add presence indicators.
  */
-export const MemberList: FC<MemberListProps> = (props: MemberListProps) => {
+export const useMemberListCore = (props: CommonMemberListProps) => {
```

Comment:
again, I don't like the file name if this is a hook from now on

Reply:
What would you propose for the file name? Hooks folder was originally for Objects API related hooks that were supposed to be moved to React SDK at some point. Not sure how else to structure this